presto-product-tests/src/main/java/com/facebook/presto/tests/jdbc/JdbcTests.java

```diff
@@ -245,13 +250,13 @@ public void testSqlEscapeFunctions()
     public void testSessionProperties()
             throws SQLException
     {
-        final String distributedJoin = "distributed_join";
```

Comment:
why this change? Can't we keep assertions for both?

Reply:
These changes look unrelated. I will remove them